accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservice/impl/AceServiceImpl.java

```diff
@@ -591,7 +592,12 @@ private String purgeAuthorizables(Set<String> authorizableIds,
         try {
             JackrabbitSession js = (JackrabbitSession) session;
             UserManager userManager = js.getUserManager();
-            userManager.autoSave(false);
+            try {
+              userManager.autoSave(false);
+            } catch (final UnsupportedRepositoryOperationException e) {
```

Comment:
Actually autoSave is always disabled with Oak, therefore it would be better to first check whether autoSave is enabled via userManager.isAutoSave() and only try to disable if that method returned true. In that case even with AEM 6.1 there would be no exception!